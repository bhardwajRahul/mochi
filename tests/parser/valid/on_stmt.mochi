on Message as m {
  print(m.text)
}
