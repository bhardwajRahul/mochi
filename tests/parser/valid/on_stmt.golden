(program
  (on Message
    (call print
      (selector text (selector m))
    )
  )
)
